openbb_platform/providers/polygon/openbb_polygon/models/stock_nbbo.py

```diff
@@ -29,36 +31,36 @@ class PolygonStockNBBOQueryParams(StockNBBOQueryParams):
             + " Splitting large requests into chunks is recommended for full-day requests of high-volume symbols."
         ),
     )
-    timestamp: Optional[Union[datetime, str]] = Field(
+    timestamp: Optional[datetime] = Field(
```

Comment:
Unfortunately, these changes break the function.  The example syntax of a bracketed function, in the PR, does not work with these changes.

```
obb.stocks.quote("SPY", provider="polygon", timestamp_gt="2023-10-26T15:00:00.000000000-04:00",timestamp_lt="2023-10-26T15:30:00.000000000-04:00")
```

```
OpenBBError: Error in Polygon request -> failed to parse timestamp query params: invalid time string: parsing time "2023-10-26 15:00:00-04:00": extra text: " 15:00:00-04:00"
```